internal/website/content/howto/pubsub/publish.md

```diff
@@ -7,40 +7,40 @@ toc: true
 
 Publishing a message to a topic with the Go CDK takes two steps:
 
-1. Open the topic with the Pub/Sub provider of your choice (once per topic).
-2. Send messages on the topic.
+1. [Open the topic][] with the Pub/Sub provider of your choice (once per topic).
+2. [Send messages][] on the topic.
```

Comment:
This is already state on the landing page for pubsub (_index.md): https://gocloud.dev/howto/pubsub/

I still like the bullets here since they provide a brief intro to the rest of the page. I don't think the TOC is a replacement